docs/_docs/02_testing-best-practices/testing-plan-output.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: collection-browser-doc
+title: Testing plan output
+category: testing-best-practices
+excerpt: >-
+  Testing specific changes described in Terraform plan output
+tags: ["testing-best-practices"]
+order: 202
```

Comment:
`202` is already used by `testing-environment.md`, so everything needs to be shifted to be correct. You don't need to change this now, but would need to be updated prior to merge.

Reply:
Yeah, I saw that, that's on my todo list.  Not sure at the moment where it would go.  I'll add a TODO comment in the file